### PR TITLE
feat(github-actions): allow configuration of max issue locks per execution

### PR DESCRIPTION
**Pull request by clydin**

### CODE REVIEW

File: github-actions/lock-closed/action.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ inputs:
   github-token:
     description: 'Github access token'
     required: true
+  max-per-execution:
```
Comment (josephperrott) [inline github-actions/lock-closed/action.yml]: Can we change this to `locks-per-execution` and then enforce a max in the code?

File: github-actions/lock-closed/src/main.ts [hunk 2]
```diff
@@ -34,6 +34,7 @@ async function run(): Promise<void> {
       `Read more about our [automatic conversation locking policy](${policyUrl}).\n\n` +
       '_This action has been performed automatically by a bot._';
 
+    const maxPerExecution = +core.getInput('max-per-execution') || 400;
```
Comment (josephperrott) [inline github-actions/lock-closed/src/main.ts]: If we do this as `locks-per-execution`:

```ts
const maxPerExecution = Math.min(+core.getInput('locks-per-execution'), 400);
```

Comment (clydin) [reply github-actions/lock-closed/src/main.ts]: Done.

File: github-actions/lock-closed/action.yml (hunk 1 above)
Comment (clydin) [reply github-actions/lock-closed/action.yml]: Done.

**Pull request merged**